--- a/configure.ac
+++ b/configure.ac
@@ -117,9 +117,9 @@ AC_SUBST(LIBNOTIFY_LIBS)
 
 # Checks AppIndicator 
 APPINDICATOR_LIBS=
-PKG_CHECK_MODULES(APPINDICATOR, appindicator3-0.1,
-     [AC_DEFINE([HAVE_APPINDICATOR],[1],[Use AppIndicator3-0.1])],
-     [AC_MSG_WARN(AppIndicator 3-0.1 not present")])
+PKG_CHECK_MODULES(APPINDICATOR, ayatana-appindicator3-0.1,
+     [AC_DEFINE([HAVE_APPINDICATOR],[1],[Use Ayatana AppIndicator3-0.1])],
+     [AC_MSG_WARN(Ayatana AppIndicator 3-0.1 not present")])
 AM_CONDITIONAL(APPINDICATOR, test -n "$APPINDICATOR_LIBS")
 AC_SUBST(APPINDICATOR_CFLAGS)
 AC_SUBST(APPINDICATOR_LIBS)
--- a/src/ui.h
+++ b/src/ui.h
@@ -27,7 +27,7 @@
 #include <gtk/gtk.h>
 
 #if defined(HAVE_APPINDICATOR)
-#include <libappindicator/app-indicator.h>
+#include <libayatana-appindicator/app-indicator.h>
 #endif
 
 #include "psensor.h"
--- a/src/ui_appindicator.c
+++ b/src/ui_appindicator.c
@@ -21,7 +21,7 @@
 #include <string.h>
 
 #include <gtk/gtk.h>
-#include <libappindicator/app-indicator.h>
+#include <libayatana-appindicator/app-indicator.h>
 
 #include <cfg.h>
 #include <psensor.h>
@@ -325,7 +325,7 @@ void ui_appindicator_init(struct ui_psen
 	APP_INDICATOR_GET_CLASS(indicator)->unfallback = unity_unfallback;
 
 	app_indicator_set_status(indicator, APP_INDICATOR_STATUS_ACTIVE);
-	app_indicator_set_attention_icon(indicator, ATTENTION_ICON);
+	app_indicator_set_attention_icon_full(indicator, ATTENTION_ICON, "");
 
 	ui_appindicator_update_menu(ui);
 }
